Source/JavaScriptCore/parser/Parser.cpp

```diff
@@ -2687,7 +2687,7 @@ template <class TreeBuilder> bool Parser<LexerType>::parseFunctionInfo(TreeBuild
 
     auto performParsingFunctionBody = [&] {
         return parseFunctionBody(context, syntaxChecker, startLocation, startColumn, functionStart, functionNameStart, parametersStart, constructorKind, expectedSuperBinding, functionBodyType, functionInfo.parameterCount);
-    };
```

Comment:
I think we should keep this semicolon because it is an assignment.
See https://webkit.org/code-style-guidelines/#other-punctuation, which has "auto a = [] { };"